apps/bfd-db-migrator/src/test/resources/db/migration/error-scenarios/bad-sql/V1__Create_beneficiary_and_claim_tables.sql

```diff
@@ -0,0 +1,1368 @@
+/*
```

Comment:
general comment on the testing migration sqls -- they seem awfully complicated for the purpose and it makes it hard to see what they are doing. can we strip them down to be minimal for the testcase?

Reply:
also they should have a comment at the top explaining what it does within the context of the test

Reply:
Yeah I wasn't super familiar with actually writing these files, so I just copied our existing ones to make my test cases; I agree they could/should be cleaned up to be clearer about what they're doing. I'll work on this.